libcore/src/IO/IniFileParser.cpp

```diff
@@ -165,14 +165,6 @@ void IniFileParser::Parse(const fs::path & iniFile)
 
 bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
 {
-    LOG_INFO("JuPedSim - JPScore");
```

Comment:
Why did you delete this? Shouldn't we just move it somewhere else?

Reply:
Is already printed in jpscore/src/main.cpp:
```    LOG_INFO("Starting JuPedSim - JPScore");
    LOG_INFO("Version {}", JPSCORE_VERSION);
    LOG_INFO("Commit id {}", GIT_COMMIT_HASH);
    LOG_INFO("Commit date {}", GIT_COMMIT_DATE);
    LOG_INFO("Build from branch {}", GIT_BRANCH);
    LOG_INFO("Build with {}({})", compiler_id, compiler_version);